### PR TITLE
Removed curator5 references from 5.1 CSV

### PR DESCRIPTION
**Pull request by vimalk78**
### Description
Curator is removed in 5.1
Removed remaining references of curator5 from CSV

/assign @jcantrill

### CODE REVIEW

File: manifests/5.1/cluster-logging.v5.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -403,19 +401,6 @@ spec:
         path: collection.logs.fluentd.nodeSelector
         x-descriptors:
         - 'urn:alm:descriptor:com.tectonic.ui:nodeSelector'
-      - description: Resource requirements for the Curator pods
```
Comment (jcantrill) [inline manifests/5.1/cluster-logging.v5.1.0.clusterserviceversion.yaml:406]: @alanconway I think this has to remain for compatibility reasons but is just no longer honored.  Maybe we just need to update the main "curation" note

**Review by alanconway — COMMENTED** [review]
/approve
/lgtm

**Review by igor-karpukhin — APPROVED** [review]
/lgtm

**Comment by rolfedh** [comment]
Please hold off on merging this PR until we've completed this conversation with Christian: https://coreos.slack.com/archives/GGUR75P60/p1620636639219100

**Comment by jcantrill** [comment]
/hold cancel

This is a cleanup of another PR that merged months ago and will unblock cpaas.  We can revert this and the other if needed

**Pull request merged**